HW1/HW1.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+let number1 = 2;
+let number2 = 25;
+
+// Addition
+let sum;
+sum = number2+number1;
+console.log("number2 + number1 = " + sum);
```

Comment:
```suggestion
console.log(`number2 + number1 = ${sum}`);
```
Тут і далі краще зразу користуватись новим синтаксисом